### DISCUSSION TITLE
How do i close a opened dropdown using JS only.

### DISCUSSION THREAD

**Discussion by baseplate-admin** [saadeghi/daisyui#1699] [Q&A] [answered]
Hi everyone,

I am facing a problem where i need to close the opened dropdown using JS only. Is there an option without toggling `dropdown-open`

**Comment by polyzen** [comment, marked as answer]
https://github.com/saadeghi/daisyui/issues/157
https://github.com/saadeghi/daisyui/issues/201

**Comment by baseplate-admin** [comment]
I think this should be added in documentation :)
